XLS-0070d-credentials/README.md

```diff
@@ -0,0 +1,595 @@
+
+It is recommended that anyone who holds a credential also have a DID object, to increase compliance with the W3C VC spec.
+
+Note that the issuer of the on-chain `Credential` does not have to be the same as the issuer of the VC, and the on-chain object can instead essentially serve as an on-chain attestation from the issuer for the VC.
+
+## 11. Invariants
+
+### 11.1. Reserves
+
+The burden of reserve should be with the issuer, and the credential should be in the issuer's owner directory, if the `lsfAccepted` flag is off. The burden of reserve should be with the subject and the credential should be in the subject's owner directory, if the `lsfAccepted` flag is on.
+
+### 11.2. The `DepositPreauth` Object
+
+A `DepositPreauth` ledger object always has exactly one of the `Authorize` field and the `AuthorizeCredentials` field.
+
+If the `AuthorizeCredentials` field is included, the array contains between 1 and 8 credentials.
+
+## 12. Examples
+
+In this example, a trusted issuer, Isabel, is issuing a credential that indicates that Alice is KYC'd. Verity is setting up her account to only interact with accounts that Isabel has attested to being properly KYC'd.
+
+For ease of reading, some of the common transaction fields, such as signatures and public keys, have been left out of this example.
+
+### 12.1. `CredentialCreate`
+
+Isabel creates the credential for Alice, after confirming her KYC status off-chain.
+
+```typescript
+{
+    TransactionType: "CredentialCreate",
+    Account: "rISABEL......",
+    Subject: "rALICE.......",
+    CredentialType: "4B5943", // "KYC" in hex
+    Expiration: 789004799, // the end of the year
+    URI: "isabel.com/credentials/kyc/alice" // This will be converted into hex
+}
+```
+
+### 12.2. The `Credential` Object
+
+This is the object created by the `CredentialCreate` transaction.
+
+```typescript
+{
+    LedgerEntryType: "Credential",
+    Flags: 0,
+    Subject: "rALICE.......",
+    Issuer: "rISABEL......",
+    CredentialType: "123ABC",
+    Expiration: 789004799, // the end of the year
+    URI: "isabel.com/credentials/kyc/alice" // This will be converted into hex
+}
+```
+
+### 12.3. `CredentialAccept`
+
+Alice accepts the credential, thereby making it valid.
+
+```typescript
+{
+    TransactionType: "CredentialAccept",
+    Account: "rALICE.......",
+    Issuer: "rISABEL......",
+    CredentialType: "123ABC"
+}
+```
+
+### 12.4. `DepositPreauth`
+
+Verity sets up her account to only interact with accounts that Isabel has KYC'd.
+
+```typescript
+{
+    TransactionType: "DepositPreauth",
+    Account: "rVERITY......",
+    AuthorizeCredentials: [
+        {
+            Credential: {
+                Issuer: "rISABEL......",
+                CredentialType: "4B5943"
```

Comment:
Suggestion: `CredentialType: "123ABC"`, Alice's credential has a different `CredentialType` than what's accepted in the DepositPreauth